### ISSUE TITLE
[$500] Workspace - Workspace creator cannot be deleted from WS after ownership change

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#43110]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.79-3
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4593414
**Email or phone of affected tester (no customers):** applausetester+47pronin@applause.expensifail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
As User A:
1. Go to https://staging.new.expensify.com/
2. Create a Workspace
3. Ad User B as a member
4. Set User B as Admin
As User B:
5. Go to created workspace
6. Transfer ownership
7. Delete Workspace creator from Member list
or 7.1 : as User A leave the workspace

## Expected Result:
As workspace creator is no longer the owner it should be possible to eliminate him from members list

## Actual Result:
Workspace creator cannot be eliminated from members list. Error is dispalyed:
"To remove [applausetester+51pronin@applause.expensifail.com](mailto:applausetester+51pronin@applause.expensifail.com)
from this workspace, please go to Admin > ANY TEST NAME WS > Connections and change the technical contact.
There was a problem removing that workspace member."
If workspace creator tries to leave the workspace by himself, then red-dot appears without any error message.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/c2f3342f-e0c4-4d7d-a820-4a288ebc03d4

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0130efde92882d3c32</li>
        <li>Upwork Job ID: 1799180402376054832</li>
        <li>Last Price Increase: 2024-07-22</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @thesahindia</details>

**Comment by kevinksullivan** [comment]
Low priority collect bug IMO

**Comment by kevinksullivan** [comment]
Going to raise the price to $500

**Comment by kevinksullivan** [comment]
hey @isabelastisser I am OOO the rest of the week, so looping in another BZ member. I'll take it back when I return!

**Comment by isabelastisser** [comment]
No proposals yet.

**Comment by isabelastisser** [comment]
No updates yet.

**Comment by kevinksullivan** [comment]
Stepping back in for @isabelastisser

**Comment by kevinksullivan** [comment]
Going to keep this at $500 and wait on proposals.

**Comment by kevinksullivan** [comment]
No update, waiting on proposals

**Comment by kevinksullivan** [comment]
Also not overdue.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by kevinksullivan** [comment]
Can no longer reproduce
